docs/data-management-on-carbondata.md

```diff
@@ -461,25 +461,46 @@ This tutorial is going to introduce all commands and data operations on CarbonDa
 
 ## COMPACTION
 
-This command merges the specified number of segments into one segment, compaction help to improve query performance.
-```
+  Compaction help to improve query performance, because frequently load data, will generate several CarbonData files, because data is sorted only within each load(per load per segment and one B+ tree index).
```

Comment:
Compaction improves the query performance significantly. During the load data, several CarbonData files are generated, this is because data is sorted only within each load (per load segment and one B+ tree index).